moped-etl/README.md

```diff
@@ -1 +1,34 @@
 # MOPED - ETL
+
+These are programs intended to be used for scheduled tasks related to transferring data into and out of Moped. These programs come in the form of Python scripts intended to be run inside a Docker container. The Dockerfile needed to build the image used for the container is included.
+
+This ETL environment is modeled after the [ATD Vision Zero ETL environment](https://github.com/cityofaustin/atd-vz-data/atd-etl).
```

Comment:
☝️ Not VZ anymore

Reply:
Can you clarify what you mean here?

Reply:
Ah, my bad. The doc looks totally fine, nvm!